toqito/matrices/tests/test_operator_sinkhorn.py

```diff
@@ -0,0 +1,148 @@
+"""Tests for operator_sinkhorn."""
+
+import warnings
+
+import numpy as np
+import pytest
+
+from toqito.channels.partial_trace import partial_trace
+from toqito.matrices.operator_sinkhorn import operator_sinkhorn
+from toqito.rand import random_density_matrix
+from toqito.states import bell
+
+
+def test_operator_sinkhorn_unitary_invariance():
+    """Test invariance of operator Sinkhorn on swapping subsystems."""
+    rho = random_density_matrix(4)
+
+    # Swap subsystems.
+    U = np.kron(np.eye(2), np.array([[0, 1], [1, 0]]))
+    rho_new = U @ rho @ U.conj().T
+    sigma_new, local_ops_new = operator_sinkhorn(rho_new)
+    sigma_old, local_ops_old = operator_sinkhorn(rho)
+    np.testing.assert_allclose(sigma_new, U @ sigma_old @ U.conj().T)
+
+
+def test_operator_sinkhorn_trace_property():
+    """Test the trace property of operator Sinkhorn when a valid inputs are passed."""
+    rho = random_density_matrix(9)
+
+    # The dimension `3` divides evenly into `9`, so no error should occur.
+    sigma, local_ops = operator_sinkhorn(rho, dim=[3])
+
+    # Check that sigma is a valid density matrix with trace equal to 1.
+    np.testing.assert_almost_equal(np.trace(sigma), 1)
```

Comment:
I prefer to not use the np testing module. Could you please use pytest assert for the unit tests in this file?